### ISSUE TITLE
using pagespace, the embeddings seems small and centralize 0

### ISSUE THREAD

**Issue by guihui** [facebookresearch/StarSpace#106]
command:
./starspace train -trainFile pagespace.train -model pagespace.model -label 'n' -trainMode 1 -dim 128 -minCount 5 -minCountLabel 5 -epoch 8 -saveTempModel true -thread 16

Data scale 30,000,000 user * 500,000 page, dim=128

I calculate the first dim histogram blew, it seems that the number is very small  and centralize 0.
I'm not sure whether the result is correct.

   -0.0457 -    -0.0373 [     4]: 
   -0.0373 -    -0.0290 [    18]: 
   -0.0290 -    -0.0206 [    87]: *
   -0.0206 -    -0.0122 [   453]: ******
   -0.0122 -    -0.0038 [  2346]: **********************************
   -0.0038 -     0.0045 [  5215]: ***************************************************************************
    0.0045 -     0.0129 [  1533]: **********************
    0.0129 -     0.0213 [   259]: ***
    0.0213 -     0.0297 [    77]: *
    0.0297 -     0.0380 [     8]:

**Comment by ledw** [comment]
@guihui Thanks for reporting. It looks like your embeddings are not well trained yet, possibly that your data is sparse. One thing you can tune is the -initRandSd which is the initialization of the embeddings. You can increase that parameter. Other things that you can tune is the learning rate and how many epochs you go through the data. Please let me know if tuning those parameters can help.

**Comment by ledw** [comment]
@guihui any updates?

**Comment by ledw** [comment]
Closing task as there's no update for a while.
